### ISSUE TITLE
Missing Hover Labels for UI Buttons

### ISSUE THREAD

**Issue by tnaum-ms** [microsoft/vscode-documentdb#90]
Some interactive buttons are missing hover labels or tooltips, reducing accessibility and usability. These should be added to improve clarity.

The WebView 'CollectionsView' contains buttons in a toolbar, for document operations:
- new document
- view document
- edit document
- delete document

They lack tooltips. Use the tooltip element from Fluent, that's used elsewhere, and add tooltips.

**Comment by tnaum-ms** [comment]
`selcukbozcan`
